From 822476e3f8128c8a8e68e2dc18b8369b692b8b82 Mon Sep 17 00:00:00 2001
From: Vinu Vaghasia <vinu.vaghasia@amd.com>
Date: Tue, 21 Jun 2022 17:00:24 -0500
Subject: [PATCH 1/1] linux-aspeed: Modify disabled MCTP header data_dump with
 aspeed-mctp driver

Disable MCTP Header debug data_dump call, we don't need with normal run
operation

Signed-off-by: Vinu Vaghasia <vinu.vaghasia@amd.com>
---
 drivers/soc/aspeed/aspeed-mctp.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/soc/aspeed/aspeed-mctp.c b/drivers/soc/aspeed/aspeed-mctp.c
index 58d0857c4175..7eb475f3b7cd 100644
--- a/drivers/soc/aspeed/aspeed-mctp.c
+++ b/drivers/soc/aspeed/aspeed-mctp.c
@@ -490,7 +490,7 @@ static void aspeed_mctp_emit_tx_cmd(struct mctp_channel *tx,
 		sizeof(packet->data.hdr) / sizeof(u32);
 	u32 offset;

-	data_dump(priv, &packet->data);
+	//data_dump(priv, &packet->data);
 	aspeed_mctp_swap_pcie_vdm_hdr(&packet->data);

 	if (priv->match_data->vdm_hdr_direct_xfer) {
@@ -808,7 +808,7 @@ static void aspeed_mctp_rx_tasklet(unsigned long data)
 			} else {
 				dev_dbg(priv->dev, "Failed to allocate RX packet\n");
 			}
-			data_dump(priv, &rx_packet->data);
+			//data_dump(priv, &rx_packet->data);
 			*hdr = 0;
 			rx->wr_ptr = (rx->wr_ptr + 1) % rx->buffer_count;
 			hdr = (u32 *)&rx_buf[rx->wr_ptr];
@@ -851,7 +851,7 @@ static void aspeed_mctp_rx_tasklet(unsigned long data)
 			dev_dbg(priv->dev,
 				"rx->wr_ptr = %d, rx_cmd->rx_lo = %08x",
 				rx->wr_ptr, *hdr);
-			data_dump(priv, &rx_packet->data);
+			//data_dump(priv, &rx_packet->data);
 			*hdr = 0;
 			rx->wr_ptr = (rx->wr_ptr + 1) % rx->buffer_count;
 			payload = (u32 *)&rx_buf[rx->wr_ptr];
--
2.17.1

